Automattic/jetpack
# Update video uploader preview poster image selection notice

Fixes #31988

## Proposed changes:

The notice previously read "Use the slider to choose a poster image", but there are cases where the video preview doesn't show up, perhaps due to some kind of latency making the notice somewhat confusing, as shown in the screenshot in the issue comment.

This PR updates the notice to further clarify that you can also change the poster image from the Block Settings sidebar under the Poster section:

![videopress-poster-image-selection-notice-update](https://github.com/Automattic/jetpack/assets/50293775/3d05ff53-2dd3-4331-aee4-0075d2cba4b7)

### Other information:


## Does this pull request change what data or activity we track or use?

No, it does not.

## Testing instructions:

1. Build the VideoPress package and activate the plugin.
2. Add the VideoPress block to a post or page.
3. Upload a video.
4. Notice the updated upload/processing preview and poster image selection notice.
